### PR TITLE
Fix kustomize binary path in Tiltfile

### PR DESCRIPTION
**Pull request by Xenwar**
Fix kustomize binary path in Tiltfile

### CODE REVIEW

File: Tiltfile [hunk 1]
```diff
@@ -162,5 +162,5 @@ validate_auth()
 include_user_tilt_files()
 
 load_provider_tiltfiles(["."])
-local("make bin/kustomize")
+local("make tools/bin/kustomize")
```
Comment (furkatgofurov7) [inline Tiltfile:165]: This is indeed a wrong path https://github.com/metal3-io/baremetal-operator/blob/master/Makefile#L21

**Review by furkatgofurov7 — COMMENTED** [review]
/lgtm

/cc @andfasano @fmuyassarov @dtantsur

**Comment by furkatgofurov7** [comment]
cc @zaneb

**Review by fmuyassarov — APPROVED** [review]

**Pull request merged**